app_python/app.py

```diff
@@ -0,0 +1,11 @@
+import pytz
+from datetime import datetime, timezone
+from flask import Flask
```

Comment:
Nit: According to PEP8, imports should be sorted by the locality, and their alphabetic order
```suggestion
from datetime import datetime, timezone

import pytz
from flask import Flask
```